docs/src/index.md

```diff
@@ -10,6 +10,8 @@ See [API documentation](@ref) for how to construct AlignedSpans, along with some
 
 Continuous timespans can be rounded (or "aligned") to the individual sample values by using the constructor `AlignedSpan`, which takes a `sample_rate`, a `span`, and a description of how to round time endpoints to indices. This constructs an `AlignedSpan` which supports Onda indexing. Internally, an `AlignedSpan` stores indices, not times, and any rounding happens when it is created instead of when indexing into `samples`.
 
+Note that AlignedSpans treats primarily treats samples as instants in time (rather than spans), as Onda itself does, and cares about "which samples have occurred by such and such point in time" rather than "what sample-span is ongoing at such and such point in time".
```

Comment:
maybe make this a `!!! note` to give it the emphasis it deserves?

Reply:
```suggestion
!!! note
    AlignedSpans treats primarily treats samples as instants in time (rather than spans), as Onda itself does, and cares about "which samples have occurred by such and such point in time" rather than "what sample-span is ongoing at such and such point in time".
```